src/core/blockchain/celo/config.ts

```diff
@@ -14,6 +14,39 @@ import {
     CELO_USD_TESTNET_BAKLAVA
 } from './tokens/cUSD';
 
+export const accountCTA = {
+    mainCta: {
+        title: 'App.labels.quickVote',
```

Comment:
Why not use here `title: translate('App.labels.quickVote')`,
I think it's easier to `translate` the title in config, rather than everywhere it's used.

Reply:
Understood, well it makes sense.